ProcessMaker/Nayra/Repositories/ProcessRequestRepository.php

```diff
@@ -45,17 +42,9 @@ public function create(array $transaction): Model
      */
     public function update(array $transaction): Model
     {
-        // Get the id mapped
-        $id = $this->uid2id[$transaction['id']];
-
-        // If not exists throws an error
-        if (!$id) {
-            throw new Exception("Cannot find id for uid {$transaction['id']}");
-        }
-
         // Update the request
         $properties = $transaction['properties'];
-        $model = ProcessRequest::find($id);
+        $model = ProcessRequest::find($transaction['id']);
```

Comment:
Is this right? `$transaction['id']` must be the uuid of the request, not the `id`